### PR TITLE
Add support GitHub actions

### PR DESCRIPTION
**Pull request by Smolevich**
- Add ci file for supporting GIthub Actions

### CODE REVIEW

File: .github/workflows/ci-build.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        go get -v -t -d ./...
+    - name: Instal go mod
+      run: |
+        export GO111MODULE=on
+        go mod download
+    - name: Build
+      run: go build -v .
+    - name: Test
+      run: |
+        go test -v ./fixtures
```
Comment (luza) [inline .github/workflows/ci-build.yml]: Here you can use `./...` instead of listing the subdirs.

File: .github/workflows/ci-build.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+name: Go
```
Comment (luza) [inline .github/workflows/ci-build.yml]: Name could more descriptive, something like "Building and testing against different environments".

**Comment by luza** [comment]
Thanks for the PR. Left a couple of comments.

**Comment by Smolevich** [comment]
@luza, failed tests is it ok?

**Comment by luza** [comment]
Absolutely not😅

**Comment by luza** [comment]
Ok, tests seem to be fixed. You can now refine the action.

**Comment by luza** [comment]
Can you look into this please?
https://github.com/lamoda/gonkey/runs/315691619

**Comment by Smolevich** [comment]
> Can you look into this please?
> https://github.com/lamoda/gonkey/runs/315691619
I fix it in 
https://github.com/lamoda/gonkey/pull/31

**Pull request merged**